Backend/DataAccessLayer/Database/dbContext.cs

```diff
@@ -14,12 +14,10 @@ public class DatabaseContext : DbContext
     {
         public DatabaseContext()
         {
-            this.Database.Connection.ConnectionString = "Data Source=localhost;Initial Catalog=SSO;Integrated Security=True";
+            Database.Connection.ConnectionString = "Data Source=Localhost;Initial Catalog=SSO;Integrated Security=True";
```

Comment:
Why was this change made?

Reply:
just for testing purposes when I was trying to connect to the database. I'll revert back to the original format